src/strategies/multichain/examples.json

```diff
@@ -4,58 +4,112 @@
     "strategy": {
       "name": "multichain",
       "params": {
-        "symbol": "MULTI",
+        "symbol": "STAKE",
```

Comment:
Looks good, can you just remove change on multichain examples?

Reply:
Sorry, didn't notice 😅